src/Raven.Server/Documents/Replication/ReplicationDocumentSender.cs

```diff
@@ -293,6 +296,22 @@ public bool ExecuteReplicationOnce(OutgoingReplicationStatsScope stats, ref Date
             }
         }
 
+        private TimeSpan GetDelayReplication()
+        {
+            TimeSpan delayReplicationFor = TimeSpan.Zero;
+
+            if (_parent.Destination is ExternalReplication external)
+            {
+                delayReplicationFor = external.DelayReplicationFor;
+
+                if (delayReplicationFor.Ticks > 0 && _parent._parent._server.LicenseManager.CanDelayReplication(out var _) == false)
```

Comment:
where does the user get an error if the license doesn't allow it?

Reply:
It's validated inside the `CanDelayReplication` method and raise a notification if the license doesn't include this feature (treated as any other feature that is not included in the license).